guides/v3.3.0/object-model/classes-and-instances.md

```diff
@@ -226,11 +226,15 @@ Person.create({
 
 ### Accessing Object Properties
 
-When accessing the properties of an object, use the [`get()`][7]
-and [`set()`][8] accessor methods:
+When reading a property value of an object, you can in most cases use the common Javascript dot notation, e.g. `myObject.myProperty`. 
+
+The one big exception to this rule are [Ember proxy objects][9]: if you're working with Ember proxy objects, including promise proxies for async relationships in Ember Data, you have to use Ember's [`get()`][7] accessor method to read values.
```

Comment:
Slight edit:

[Ember proxy objects][9] are the one big exception to this rule. If you're working with Ember proxy objects, including promise proxies for async relationships in Ember Data, you have to use Ember's [`get()`][7] accessor method to read values.

Reply:
I also think maybe we need a small example here.

For example, say there is an Ember Data model, `blogPost`, which has many `comment` records associated with it. We could access attributes of the `blogPost` itself using dot notation, like `blogPost.title`. However, to show the comments that are part of the relationship, we would need to use `blogPost.get('comments')`.